sql/20190428223823_create_loans_table.sql

```diff
@@ -0,0 +1,23 @@
+-- migrate:up
+CREATE TABLE "loans" (
+    "id"                   SERIAL  NOT NULL,
+    "client_id"            INT     NOT NULL,
+    "application_id"       INT     NOT NULL,
+    "amount"               INT     NOT NULL,
+    "term"                 INT     NOT NULL,
+    "interest"             INT     NOT NULL,
+    "currency"             TEXT    NOT NULL,
+    "date"                 TEXT    NOT NULL,
```

Comment:
use timestamp

Reply:
Should I fix from TEXT to TIMESTAMP in other tables?

Reply:
yeep!